diff --git a/src/components/Footer.astro b/src/components/Footer.astro
index 2f4f28f..f92f541 100644
--- a/src/components/Footer.astro
+++ b/src/components/Footer.astro
@@ -6,7 +6,6 @@
     <div class="flex flex-col md:flex-row justify-center md:justify-between items-center md:items-start gap-8">
       <div class="flex flex-col gap-2 text-sm order-3 md:order-1 text-center md:text-left">
         <span>Copyright © HacktoberfestGYE 2024</span>
-        <span>Desarrollado por: <a class="underline decoration-wavy text-muted font-bold decoration-primary" href="https://codegea.com" target="_blank" rel="noopener noreferrer">CodeGea</a></span>
       </div>
       <div class="socials flex flex-row gap-4 order-2">
         <a href="https://www.instagram.com/hacktoberfestgye" target="_blank" rel="noopener noreferrer">
